### PR TITLE
[Bug] MES-6485: Change Date of Test

### PR DESCRIPTION
**Pull request by szabikr**
### This PR features:

- [x] Increasing the width of the Date of Test input
- [x] Validating the Date of Test so that it's not in the future
- [x] Validating the Date of Test so that it's not in the past more than 12 months
- [x] Display error message and stop user from uploading incorrect data
- [x] Changing date format of `start` property to comply with the Database Schema

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by piercemorris — APPROVED** [review]

File: src/shared/helpers/__tests__/test-start-time.spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  describe('getNewTestStartTime', () => {
+    it('should return the correct date', () => {
+      const inputDate = '2021-01-19';
+      const startDateTime = '2020-12-25T08:10:00';
+      expect(getNewTestStartTime(inputDate, startDateTime)).toBe('2021-01-19T08:10:00');
+    });
+  });
+
+  describe('', () => {
```
Comment (ammarhaiderbjss) [inline src/shared/helpers/__tests__/test-start-time.spec.ts]: empty describe?

File: src/shared/helpers/__tests__/test-start-time.spec.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    const currentDate = '2021-01-22';
+
+    it('should return true when date is between current date and on year in the past', () => {
```
Comment (ammarhaiderbjss) [inline src/shared/helpers/__tests__/test-start-time.spec.ts]: typo "one"

**Review by ammarhaiderbjss — APPROVED** [review]
Looks great 👍

File: src/shared/helpers/__tests__/test-start-time.spec.ts (hunk 1 above)
Comment (szabikr) [reply src/shared/helpers/__tests__/test-start-time.spec.ts]: Ohh, the result of TDD hah

Added the name of the function under test

**Pull request merged**